contracts/Storage.sol

```diff
@@ -59,7 +63,12 @@ contract Storage is Collateral, Marketplace {
     return _missed(slotId);
   }
 
-  function isProofRequired(bytes32 slotId) public view returns (bool) {
+  function isProofRequired(bytes32 slotId)
+    public
+    view
+    slotAcceptsProofs(slotId)
```

Comment:
Why would we want to raise an error here when the request is finished?

If we do want an error, then we should probably do the same for `willProofBeRequired()`, `getChallenge()` and `getPointer()`.